tests/release/release.go

```diff
@@ -152,13 +153,14 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1108]test-release-service-happy-p
 				}
 
 				return len(pipelineRunList.Items) > 0 && err == nil
-			}, avgPipelineCompletionTime, defaultInterval).Should(BeTrue())
+			}, avgControllerQueryTimeout, defaultInterval).Should(BeTrue())
 
 			release, err := framework.ReleaseController.GetRelease(releaseName, devNamespace)
 			if err != nil {
 				klog.Error(err)
 			}
 			Expect(release.Status.ReleasePipelineRun == (fmt.Sprintf("%s/%s", pipelineRunList.Items[0].Namespace, pipelineRunList.Items[0].Name))).Should(BeTrue())
-		})
+		}, SpecTimeout(avgControllerQueryTimeout*2+namespaceDeletionTimeout*2))
+		// Consider also the AfterAll block deleting two namespaces
```

Comment:
I'm confused what this comment is saying

Reply:
I am doubling this time threshold to stay on the safe side

Reply:
Removed the comments